lodash-plugin/src/test/java/com/github/underscore/lodash/StringTest.java

```diff
@@ -1518,53 +1518,53 @@ public void chain() {
         $.chain(new String[] {""}).rest();
         $.chain(new String[] {""}).rest(1);
         $.chain(new String[] {""}).flatten();
-        $.chain(new Integer[] {0}).map(new Function1<Integer, Integer>() {
+        $.chain(new Integer[] {0}).map(new Function<Integer, Integer>() {
             public Integer apply(Integer value) { return value; } });
         $.chain(new String[] {""}).filter(new Predicate<String>() {
-            public Boolean apply(String str) { return true; } });
+            public boolean test(String str) { return true; } });
         $.chain(new String[] {""}).reject(new Predicate<String>() {
-            public Boolean apply(String str) { return true; } });
+            public boolean test(String str) { return true; } });
         $.chain(new String[] {""}).reduce(new FunctionAccum<String, String>() {
             public String apply(String accum, String str) { return null; } }, "");
         $.chain(new String[] {""}).reduceRight(new FunctionAccum<String, String>() {
             public String apply(String accum, String str) { return null; } }, "");
         $.chain(new String[] {""}).find(new Predicate<String>() {
-            public Boolean apply(String str) { return true; } });
+            public boolean test(String str) { return true; } });
         $.chain(new String[] {""}).findLast(new Predicate<String>() {
-            public Boolean apply(String str) { return true; } });
+            public boolean test(String str) { return true; } });
         $.chain(new Integer[] {0}).max();
-        $.chain(new Integer[] {0}).max(new Function1<Integer, Integer>() {
+        $.chain(new Integer[] {0}).max(new Function<Integer, Integer>() {
             public Integer apply(Integer value) { return value; } });
         $.chain(new Integer[] {0}).min();
-        $.chain(new Integer[] {0}).min(new Function1<Integer, Integer>() {
+        $.chain(new Integer[] {0}).min(new Function<Integer, Integer>() {
```

Comment:
\[FindBugs\] INFO: RV: Return value of $$Chain\.min\(Function\) ignored, is this OK in com\.github\.underscore\.lodash\.StringTest\.chain\(\)